.github/workflows/validation.yml

```diff
@@ -9,6 +9,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: "true"
+          fetch-depth: 0
```

Comment:
Why is depth set to 0?

Reply:
It's recommended in Nx Githib actions guide, without this affected command will throw error because develop branch is not fetched